client/src/views/client-area/DashboardPage.vue

```diff
@@ -1,18 +1,520 @@
+            </div>
+          </div>
+
+          <!-- storage -->
+          <div class="month-summary-item">
+            <ion-title class="month-summary-item__title title-h4">
+              {{ $msTranslate('clientArea.dashboard.summary.storage') }}
+            </ion-title>
+            <div
+              class="month-summary-item__data title-h1-xl"
+              v-if="stats"
+            >
+              {{ stats?.dataSize }}
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- recent invoices -->
+      <div class="dashboard-section invoices-container">
+        <ion-title class="dashboard-section-title title-h2">
+          {{ $msTranslate('clientArea.dashboard.invoices.title') }}
+        </ion-title>
+        <div
+          class="invoices"
+          v-if="invoices"
+        >
+          <div class="invoices-header">
+            <ion-list class="invoices-header-list ion-no-padding">
+              <ion-item class="invoices-header-list-item invoices-date">
+                <ion-text class="menu-active">{{ $msTranslate('clientArea.dashboard.invoices.date') }}</ion-text>
+              </ion-item>
+              <ion-item class="invoices-header-list-item invoices-organization">
+                <ion-text class="menu-active">{{ $msTranslate('clientArea.dashboard.invoices.organization') }}</ion-text>
+              </ion-item>
+              <ion-item class="invoices-header-list-item invoices-amount">
+                <ion-text class="menu-active">{{ $msTranslate('clientArea.dashboard.invoices.price.title') }}</ion-text>
+              </ion-item>
+              <ion-item class="invoices-header-list-item invoices-status">
+                <ion-text class="menu-active">{{ $msTranslate('clientArea.dashboard.invoices.status') }}</ion-text>
+              </ion-item>
+            </ion-list>
+          </div>
+
+          <div class="invoices-content">
+            <ion-list
+              class="invoices-list ion-no-padding"
+              v-if="invoices.length > 0"
+            >
+              <ion-item
+                class="invoices-list-item ion-no-padding"
+                v-for="invoice in invoices"
+                :key="invoice.id"
+              >
+                <ion-text class="invoices-list-item__data invoices-date subtitles-sm">{{ invoice.start.toFormat('LLLL yyyy') }}</ion-text>
+                <ion-text class="invoices-list-item__data invoices-organization body">{{ invoice.organizationId }}</ion-text>
+                <ion-text class="invoices-list-item__data invoices-amount body">{{ invoice.total }}</ion-text>
+                <ion-text class="invoices-list-item__data invoices-status">
+                  <span class="badge-status body-sm incoming">{{ invoice.status }}</span>
+                  <ion-text class="custom-button custom-button-ghost button-medium">
+                    <ion-icon :icon="download" />
+                    {{ $msTranslate('clientArea.dashboard.invoices.download') }}
```

Comment:
`Download` button does nothing. Also remember to check on electron if the PDF are downloadable.